### ISSUE TITLE
Waiting for download...

### ISSUE THREAD

**Issue by WebworkrNet** [mediathekview/zapp#344]
Video files with status "Waiting for download...": Downloading does not start even if there is sufficient storage space and an online connection. If you delete the entry from the list of downloads and start the download again, it will begin.

**Comment by WebworkrNet** [comment]
Maybe related to #309.

**Comment by cemrich** [comment]
It may take a while (up to a couple of minutes) for the downloader to react to a changed connection status.

**Comment by WebworkrNet** [comment]
I have tested specifically. After more than 10 minutes, I stopped the test.

Is this dependent on the number of "open" downloads?

**Comment by cemrich** [comment]
Yes, it may be. The default executor does allow 2 to 4 parallel work pieces (downloads in Zapps case) - depending on the devices processor count. That's okay. It does not make a lot of sense to run 10 downloads in parallel.

**Comment by WebworkrNet** [comment]
I notice a difference between younger and older waiting downloads. Younger ones regularly try to restart the download process.
